bootstrap_cfn/config.py

```diff
@@ -356,6 +357,7 @@ def rds(self):
             AutoMinorVersionUpgrade=False,
             VPCSecurityGroups=[GetAtt(database_sg, "GroupId")],
             DBSubnetGroupName=Ref(rds_subnet_group),
+            DBInstanceIdentifier=identifier,
```

Comment:
I don't think we should need to specify this. From <http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-rds-database-instance.html#cfn-rds-dbinstance-dbinstanceidentifier>

> **DBInstanceIdentifier**
>
>    A name for the DB instance. If you don't specify a name, AWS CloudFormation generates a unique physical ID and uses that ID for the DB instance. For more information, see Name Type.
>
>    _Required: No_